lib/ecs_deploy/instance_fluctuation_manager.rb

```diff
@@ -66,24 +66,22 @@ def decrease
 
       target_container_instances = extract_target_container_instances(decrease_count, az_to_container_instances)
 
-      threads = []
+      @logger.info("running tasks: #{ecs_client.list_tasks(cluster: @cluster).task_arns.size}")
       all_running_task_arns = []
       target_container_instances.map(&:container_instance_arn).each_slice(MAX_UPDATABLE_ECS_CONTAINER_COUNT) do |arns|
+        @logger.info(arns)
         ecs_client.update_container_instances_state(
           cluster: @cluster,
           container_instances: arns,
           status: "DRAINING"
         )
         arns.each do |arn|
-          threads << Thread.new(arn) do |a|
-            all_running_task_arns.concat(stop_tasks(a))
-          end
+          all_running_task_arns.concat(list_running_task_arns)
```

Comment:
`list_running_task_arns` requires an argument.

```suggestion
          all_running_task_arns.concat(list_running_task_arns(a))
```